src/test/java/org/elasticsearch/test/ElasticsearchTestCase.java

```diff
@@ -168,7 +168,20 @@ public static boolean awaitBusy(Predicate<?> breakPredicate) throws InterruptedE
     }
 
     public static boolean awaitBusy(Predicate<?> breakPredicate, long maxWaitTime, TimeUnit unit) throws InterruptedException {
+        return awaitBusy(breakPredicate, maxWaitTime, 0, unit);
+    }
+
+    /**
+     * Repeat a task until it returns true or after a given wait time.
+     * By default, it will increase by 2 the sleep time between two runs
+     * @param breakPredicate test you want to run
+     * @param maxWaitTime maximum time you want to wait
+     * @param maxSleepTime if >0 maximum sleep time before two runs
+     * @param unit time unit used for maxWaitTime and maxSleepTime
+     */
+    public static boolean awaitBusy(Predicate<?> breakPredicate, long maxWaitTime, long maxSleepTime, TimeUnit unit) throws InterruptedException {
```

Comment:
I think I would just hardcode the parameter to a reasonable value (eg. 1s) instead of making the API more complex with a new parameter?